src/main/java/uk/gov/companieshouse/siccode/api/search/SicCodeService.java

```diff
@@ -33,7 +41,21 @@ public SicCodeService(CombinedSicActivitiesRepository combinedSicActivitiesRepos
             criteria = new SicCodeSearchTextCriteria(searchString).getTextCriteriaMatchAny();
         }
 
-        return combinedSicActivitiesRepository.findAllByOrderByScore(criteria);
+        var combinedSicActivityOrderedResults = combinedSicActivitiesRepository.findAllByOrderByScore(criteria);
+
+        LOG.infoContext(sicCodeSearchRequestApiModel.getContextId(), "Search Complete", dataMap(sicCodeSearchRequestApiModel,combinedSicActivityOrderedResults));
+
+        return combinedSicActivityOrderedResults;
+    }
+
+    private Map<String, Object> dataMap(SicCodeSearchRequestApiModel sicCodeSearchRequestApiModel,
+            List<CombinedSicActivitiesStorageModel> combinedSicActivityOrderedResults) {
+                Map<String, Object> result = new LinkedHashMap<>();
```

Comment:
Why use LinkedHashMap rather than HashMap here?

Reply:
changed

Reply:
Using a linked hashmap means in the fields in the logs they are guaranteed to be in the same order. 
Your call?

Reply:
Fine as is.